proto/src/ground/v1beta1/survey.proto

```diff
@@ -85,6 +85,24 @@ message Survey {
 
   // Defines the survey's current state.
   State state = 7;
+
+  // Defines the survey's current visibility.
+  enum Visibility {
```

Comment:
Not sure "visibility" is the right name here - it's really who can collect data beyond those in ACLs. Let's just mirror Doc & Drive: `GeneralAccess` and `general_access`?